<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="code_text">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>441</width>
     <height>561</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>codes</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>561</width>
     <height>541</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>511</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>增加一组热键</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="over_it">
            <property name="text">
             <string>覆盖原有功能</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>$模式(实现中)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="keymap"/>
          </item>
          <item>
           <widget class="QLineEdit" name="hot_key_is">
            <property name="placeholderText">
             <string>热键名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hotkey_ok">
            <property name="text">
             <string>ok</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="run">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>运行程序    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="run_prog_is">
          <property name="placeholderText">
           <string>程序名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="run_prog_ok">
          <property name="text">
           <string>ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Msg">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>弹出消息框</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Msg_info">
          <property name="placeholderText">
           <string>内容</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Msg_ok">
          <property name="text">
           <string>ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Msg_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>传值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>内容  ：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Msg_info_2">
          <property name="placeholderText">
           <string>被传值的变量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>&lt;==</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Msg_info_3">
          <property name="placeholderText">
           <string>值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Msg_ok_2">
          <property name="text">
           <string>ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>发送按键</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="key2send">
          <property name="placeholderText">
           <string>要发送的按键</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="key_mode"/>
        </item>
        <item>
         <widget class="QSpinBox" name="time_show"/>
        </item>
        <item>
         <widget class="QPushButton" name="sendkeyok">
          <property name="text">
           <string>ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QPushButton" name="t_btn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>t</string>
     </property>
    </widget>
    <widget class="QPushButton" name="color_it">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>上色</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>haha</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>395</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="setting">
      <property name="text">
       <string>settings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>save as</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="about_btn">
      <property name="text">
       <string>about</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="run_btn">
      <property name="text">
       <string>run</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>1051</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
